### PR TITLE
Update CHANGELOG.MD

### PR DESCRIPTION
**Pull request by Bataran**
Update CHANGELOG.md for 1.279.1 and future 1.289.1

Note: Also added .idea to the .gitignore

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -1,3 +1,22 @@
+## 1.289.1 (2024-04-17)
+
+### Features
```
Comment (Oberon00) [inline CHANGELOG.MD:3]: These are not features, I would call these "Fixes" and mention the trigger but also remark that we are not actually affected.
Maybe a single point "Update dependency versions" would be enough? Details can be seen in the diffs anyways if needed.

File: CHANGELOG.MD [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+* Update go version to 1.17
+* Update google.golang.org/protobuf version to 1.33.0.
+
+## 1.279.1 (2023-12-21)
+
+### Features
+
+* Improve reading config file
+* Split large spans into multiple export messages
+* Update semantic constants
```
Comment (Oberon00) [inline CHANGELOG.MD:16]: This & the next 2 points don't really mean anything to users, most of them are internal improvements for compatibility with future server-side features.

File: CHANGELOG.MD [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+* Update gopkg.in/yaml.v3 version to 3.0.0.
+* Update go version to 1.17
+* Update google.golang.org/protobuf version to 1.33.0.
+
+## 1.279.1 (2023-12-21)
+
+### Features
+
+* Improve reading config file
+* Split large spans into multiple export messages
```
Comment (Oberon00) [inline CHANGELOG.MD:15]: Spans are never split, only multiple spans are distributed over multiple messages.
Maybe user-facing we could say "Improve reliability when sending large traces"

File: CHANGELOG.MD [hunk 4]
```diff
@@ -1,3 +1,22 @@
...
+* Update golang.org/x/sys version to 0.1.0.
+* Update gopkg.in/yaml.v3 version to 3.0.0.
+* Update go version to 1.17
+* Update google.golang.org/protobuf version to 1.33.0.
+
+## 1.279.1 (2023-12-21)
+
+### Features
+
+* Improve reading config file
```
Comment (Oberon00) [inline CHANGELOG.MD:14]: How so?

File: CHANGELOG.MD (hunk 1 above)
Comment (Bataran) [reply CHANGELOG.MD:10]: I changed this, please check again

File: CHANGELOG.MD (hunk 2 above)
Comment (Bataran) [reply CHANGELOG.MD:21]: Shouldn't we also put changes useful for developers only

File: CHANGELOG.MD (hunk 3 above)
Comment (Bataran) [reply CHANGELOG.MD]: Thanks, I'll use your suggestion.

Note: I was not working on the changes so I don't know what exactly changed , I'm just trying to document based on commits.

File: CHANGELOG.MD (hunk 4 above)
Comment (Bataran) [reply CHANGELOG.MD]: I'll add more details on this

**Review by Ma-Ba — DISMISSED** [review]

File: CHANGELOG.MD (hunk 2 above)
Comment (Oberon00) [reply CHANGELOG.MD:21]: These are only useful for developers at Dynatrace, not outside developers, or do I misunderstand something?

**Review by Oberon00 — APPROVED** [review]

File: CHANGELOG.MD (hunk 2 above)
Comment (Bataran) [reply CHANGELOG.MD:21]: Yes, but I don't think we'll have many (if any) external contributors here.

**Review by Ma-Ba — APPROVED** [review]

**Pull request merged**